python/sbp/client/drivers/network_drivers.py

```diff
@@ -97,11 +111,41 @@ class HTTPDriver(BaseDriver):
 
   def __init__(self,
                device_uid,
-               url="http://broker.testing.skylark.swiftnav.com"):
+               url="http://broker.testing.skylark.swiftnav.com",
+               timeout=DEFAULT_TIMEOUT):
+    self.session = requests.Session()
+    retry = Retry(connect=MAX_CONNECT_RETRIES,
+                  read=MAX_READ_RETRIES,
+                  redirect=MAX_REDIRECTS,
+                  status_forcelist=[500],
+                  backoff_factor=DEFAULT_BACKOFF_FACTOR)
+    self.session.mount("http://",
+                       HTTPAdapter(pool_connections=DEFAULT_POOLSIZE,
```

Comment:
I will note, the HTTPAdapter that configures retries actually already has a connection pool. My motivation here was to make that explicit in case we wanted to change it.